cluster/manifests/prometheus/configmap.yaml

```diff
@@ -10,7 +10,8 @@ data:
     global:
       scrape_interval: 15s
       external_labels:
-        monitor: 'skipper-ingress'
+        pod_name: @@POD_NAME@@
```

Comment:
Maybe call this a more generic name like `replica` as `pod_name` would be a valid label name in metrics?